### DISCUSSION TITLE
Deploying SO installation with bridge interface for inline IDS ?

### DISCUSSION THREAD

**Discussion by KyungWonPark** [Security-Onion-Solutions/securityonion#8366] [Unsupported Versions] [answered]
Hello

I want to deploy SO with transparent bridge
I'd like to bridge two interfaces: eth0, eth1 and make SO monitor eth0
But it seems like SO can only select unmanaged interfaces at installation step
Also bridge interface does not show up

Do I must have switch with port mirroring capability to use SO?
Thanks

**Comment by isaacgolding** [comment, marked as answer]
SO isn't going to present you with the option to use bridges or other options.   It needs 1 management interface and at least 1 sensor interface.   The interfaces can be real or they can be virtual (as in a "virtual machine").    You can plug the sensor interface into a managed switch or an unmanaged switch or hub or a dedicated hardware network tap (or build a virtual setup the sensor would "plug" into).  

Ideally the best option is a dedicated hardware tap or managed switch with some type of mirror or span capability.  And that's what you really want in a mission critical lan.   But for educational/testing/lab/low $ environments the virtual or unmanaged setups are not bad at all.

My personal setup is a Proxmox VM setup with 4 separate physical sensors going into 4 separate switches.  Each nic is setup in proxmox to be plugged into virtual traditional hubs so that the VM sees all of the traffic that the proxmox host can see.   But all of that confiuration is done on the proxmox side.      The SO setup is simply just telling SO that these 4 ports are to be used as sesnors and SO installer does what it wants to do to them.

From the docs:  

> NIC
> You’ll need at least one wired network interface dedicated to management (preferably connected to a dedicated management network). We recommend using static IP addresses where possible.
> 
> If you plan to sniff network traffic from a tap or span port, then you will need one or more interfaces dedicated to sniffing (no IP address). The installer will automatically disable NIC offloading functions such as tso, gso, and gro on sniffing interfaces to ensure that [Suricata](https://docs.securityonion.net/en/2.3/suricata.html#suricata) and [Zeek](https://docs.securityonion.net/en/2.3/zeek.html#zeek) get an accurate view of the traffic.
> 
> Make sure you get good quality network cards, especially for sniffing. Most users report good experiences with Intel cards.
> 
> Security Onion is designed to use wired interfaces. You may be able to make wireless interfaces work, but we don’t recommend or support it.
> 

https://docs.securityonion.net/en/2.3/hardware.html?highlight=TAP%20device
